app/models/case_contact.rb

```diff
@@ -21,10 +19,40 @@ class CaseContact < ApplicationRecord
   ].freeze
 
   CONTACT_MEDIUMS = %w[in-person text/email video voice-only letter].freeze
-  enum contact_type: CONTACT_TYPES.zip(CONTACT_TYPES).to_h
+  # enum contact_type: CONTACT_TYPES.zip(CONTACT_TYPES).to_h
+
+  validates :contact_types, presence: true
+  validate :contact_types_included
+
+  def contact_types_included
+    contact_types&.each do |contact_type|
+      errors.add(:contact_types, :invalid) unless CONTACT_TYPES.include? contact_type
+    end
+  end
+
+
+  # @return [Array<String>]
```

Comment:
Cool, types! How is this used? Is it an IDE plugin?

Reply:
Yep for vscode.